frontend/src/static/js/v_summary.js

```diff
@@ -44,9 +44,10 @@ window.vSummary = {
     return {
       filtered: [],
       filterSearch: '',
-      filterSort: 'displayName',
       filterSortReverse: false,
       filterGroupSelection: 'groupByRepos',
+      sortGroupSelection: 'searchPath asc',
+      sortWithinGroupSelection: 'name asc',
```

Comment:
not setting the options in here?

Reply:
`getOptionWithOrder` is called within `sortFiltered`, which will be called upon initiation.

Reply:
as in the intialization should be done here as well, just put an empty string

Reply:
Okay added.